pkg/daemon/ceph/client/radosnamespace.go

```diff
@@ -104,3 +104,32 @@ func DeleteRadosNamespace(context *clusterd.Context, clusterInfo *ClusterInfo, p
 	logger.Infof("successfully deleted rados namespace %s/%s in k8s namespace %q", poolName, namespaceName, clusterInfo.Namespace)
 	return nil
 }
+
+// EnableRBDRadosNamespaceMirroring enables rbd mirroring on a rados namespace.
+func EnableRBDRadosNamespaceMirroring(context *clusterd.Context, clusterInfo *ClusterInfo, poolAndRadosNamespaceName, remoteNamespace, mode string) error {
```

Comment:
can we move these two functions to mirror.go where we have all the helper mirroring functions?